static/js/util/sorting.js

```diff
@@ -1,26 +1,35 @@
 // @flow
 import moment from 'moment';
+import _ from 'lodash';
 
-import type { WorkHistoryEntry } from '../flow/profileTypes';
+import type { WorkHistoryEntry, EducationEntry } from '../flow/profileTypes';
 
-export function resumeOrder(entries: WorkHistoryEntry[], dateFieldName: string): WorkHistoryEntry[] {
-  let sortFunc = (a, b) => {
-    let adate = moment(a[dateFieldName]);
-    let bdate = moment(b[dateFieldName]);
-    if ( adate.isBefore(bdate) ) {
-      return 1;
-    }
-    if ( adate.isAfter(bdate) ) {
-      return -1;
-    }
-    return 0;
+export function momentSortDesc(a: moment$Moment, b: moment$Moment): number {
```

Comment:
this function isn't actually sorting. it acts as a comparator. it's a small enough function to read, but all the same i would prefer not to have a misleading name. `momentDescComparator` et al